cypress/support/iterator.js

```diff
@@ -0,0 +1,41 @@
+import config from './config/services';
+import alwaysTests from '../integration/pages';
+
+const serviceHasPageType = (service, pageType) =>
```

Comment:
I wrote the [exact same function](https://github.com/bbc/simorgh/blob/0a67dc52c6da8331d28261c2f7cf833b94d34578/cypress/support/serviceHasPageType/index.js) but it's curried like this:

```js
const serviceHasPageType = pageType => service =>
  pathSatisfies(
    Boolean,
    [service, 'pageTypes', pageType, 'path'],
    servicesConfig,
  );
```

enabling you to do this

```js
.filter(serviceHasPageType(pageType))
```

instead of this

```js
.filter(service => serviceHasPageType(service, pageType))
```

Reply:
Gunna leave it as is, lower barrier for entry IMO